src/pudl/analysis/state_demand.py

```diff
@@ -117,404 +70,6 @@ def lookup_state(state: str | int) -> dict:
     return {x[key].lower(): x for x in STATES}[state.lower()]
 
 
-def local_to_utc(local: pd.Series, tz: Iterable, **kwargs: Any) -> pd.Series:
-    """Convert local times to UTC.
-
-    Args:
-        local: Local times (tz-naive ``datetime64[ns]``).
-        tz: For each time, a timezone (see :meth:`DatetimeIndex.tz_localize`)
-            or UTC offset in hours (``int`` or ``float``).
-        kwargs: Optional arguments to :meth:`DatetimeIndex.tz_localize`.
-
-    Returns:
-        UTC times (tz-naive ``datetime64[ns]``).
-
-    Examples:
-        >>> s = pd.Series([pd.Timestamp(2020, 1, 1), pd.Timestamp(2020, 1, 1)])
-        >>> local_to_utc(s, [-7, -6])
-        0   2020-01-01 07:00:00
-        1   2020-01-01 06:00:00
-        dtype: datetime64[ns]
-        >>> local_to_utc(s, ['America/Denver', 'America/Chicago'])
-        0   2020-01-01 07:00:00
-        1   2020-01-01 06:00:00
-        dtype: datetime64[ns]
-    """
-    return local.groupby(tz, observed=True).transform(
-        lambda x: x.dt.tz_localize(
-            datetime.timezone(datetime.timedelta(hours=x.name))
-            if isinstance(x.name, int | float)
-            else x.name,
-            **kwargs,
-        ).dt.tz_convert(None)
-    )
-
-
-def utc_to_local(utc: pd.Series, tz: Iterable) -> pd.Series:
-    """Convert UTC times to local.
-
-    Args:
-        utc: UTC times (tz-naive ``datetime64[ns]`` or ``datetime64[ns, UTC]``).
-        tz: For each time, a timezone (see :meth:`DatetimeIndex.tz_localize`)
-          or UTC offset in hours (``int`` or ``float``).
-
-    Returns:
-        Local times (tz-naive ``datetime64[ns]``).
-
-    Examples:
-        >>> s = pd.Series([pd.Timestamp(2020, 1, 1), pd.Timestamp(2020, 1, 1)])
-        >>> utc_to_local(s, [-7, -6])
-        0   2019-12-31 17:00:00
-        1   2019-12-31 18:00:00
-        dtype: datetime64[ns]
-        >>> utc_to_local(s, ['America/Denver', 'America/Chicago'])
-        0   2019-12-31 17:00:00
-        1   2019-12-31 18:00:00
-        dtype: datetime64[ns]
-    """
-    if utc.dt.tz is None:
-        utc = utc.dt.tz_localize("UTC")
-    return utc.groupby(tz, observed=True).transform(
-        lambda x: x.dt.tz_convert(
-            datetime.timezone(datetime.timedelta(hours=x.name))
-            if isinstance(x.name, int | float)
-            else x.name
-        ).dt.tz_localize(None)
-    )
-
-
-# --- Datasets: References --- #
-
-
-def load_ventyx_hourly_state_demand(path: str) -> pd.DataFrame:
-    """Read and format Ventyx hourly state-level demand.
```

Comment:
This function for checking our outputs seems to have disappeared. Is that intentional?